pmlc/conversion/tile_to_pxa/tile_to_pxa.cc

```diff
@@ -1144,10 +1165,10 @@ struct LowerTileToPXAPass : public LowerTileToPXABase<LowerTileToPXAPass> {
                                 ResultIs<EltwiseInteger>>,
         ContractionOpConversion<CombinationKind::eq,
                                 CmpFloatOp<CmpFPredicate::OEQ>,
-                                ResultIs<EltwiseFloat>>,
+                                AnyComparandIs<EltwiseFloat>>,
         ContractionOpConversion<CombinationKind::eq,
                                 CmpIntOp<CmpIPredicate::eq>,
-                                ResultIs<EltwiseInteger>>,
+                                ComparandsAre<EltwiseInteger>>,
```

Comment:
Does this mean we no longer have a conversion for heterogenous operand types?

Reply:
The previous ContractionOpConversion should cover that one, using CmpFloatOp. The issue was that when we have heterogenous operand types, they are promoted to float, not to int.

Reply:
OK, sorry, took me a second to understand, LGTM.